### PR TITLE
engines/fileoperations: add more description for file/directory opera…

### PR DESCRIPTION
**Pull request by friendy-su**
There are some QA for file/directory operation engines, like https://github.com/axboe/fio/issues/1699
This patch adds more descriptions about these new engines into HOWTO.rst.

### CODE REVIEW

File: HOWTO.rst [hunk 1]
```diff
@@ -2276,6 +2248,50 @@ I/O engine
...
+			Example job file: dirstat-ioengine.fio.
+
+		**dirdelete**
+			Simply delete the directories by rmdir() and do no I/O to them. You need to set 'filesize'
+			and 'nrfiles', so that the directories will be created.
+			This engine is to measure directory delete.
+			Example job file: dirdelete-ioengine.fio.
+
+		For file and directory operation engines, there is no I/O throughput, then the
+		statistics data in report have different meanings. The meaningful output indexes are: 'iops', 'clat'.
```
Comment (vincentkfu) [inline HOWTO.rst:2291]: Please replace the comma between iops and clat with 'and'

**Comment by vincentkfu** [comment]
Please make similar changes to fio.1

File: HOWTO.rst (hunk 1 above)
Comment (friendy-su) [reply HOWTO.rst]: added 'and'

**Comment by friendy-su** [comment]
> Please make similar changes to fio.1

add same content to fio.1.

**Pull request merged**